src/Sulu/Bundle/CategoryBundle/Resources/public/js/components/categories/edit/details/main.js

```diff
@@ -134,12 +148,21 @@ define(['text!./form.html'], function(form) {
          * starts editable list for keywords.
          */
         startKeywordList: function() {
+            var $toolbar = $('<div/>'),
+                $list = $('<div/>');
+
+            this.sandbox.stop(this.$find('#keywords-list *'));
+            this.sandbox.stop(this.$find('#keywords-list-toolbar *'));
```

Comment:
This should actually be done when the route changes... Why is this not happening? Is the backbone router called with `trigger: false` here? Shouldn't we change that, to be consistent with all the other bundles instead of stopping stuff manually?